ui/app/components/clients/table.ts

```diff
@@ -49,24 +49,37 @@ interface Args {
 
 export default class ClientsTable extends Component<Args> {
   @tracked currentPage = 1;
-  @tracked pageSize = Ember.testing ? 3 : 10; // lower in tests to test pagination without seeding more data
```

Comment:
3 is not a supported page size when the size selector renders soo got rid of this for simplicty